src/main/vcpf4/usb_conf.h

```diff
@@ -160,13 +160,21 @@
 
 /****************** USB OTG FS CONFIGURATION **********************************/
 #ifdef USB_OTG_FS_CORE
+#ifdef USB_CDC_HID
+ #define RX_FIFO_FS_SIZE                          128
+ #define TX0_FIFO_FS_SIZE                          32
+ #define TX1_FIFO_FS_SIZE                          32
+ #define TX2_FIFO_FS_SIZE                          32
+ #define TX3_FIFO_FS_SIZE                          64
+#else
  #define RX_FIFO_FS_SIZE                          128
  #define TX0_FIFO_FS_SIZE                          64
  #define TX1_FIFO_FS_SIZE                         128
  #define TX2_FIFO_FS_SIZE                          0
  #define TX3_FIFO_FS_SIZE                          0
  #define TXH_NP_FS_FIFOSIZ                         96
```

Comment:
These are different for CDC+USB?

Reply:
Yep, larger FIFO allows for bigger packets on VCP

Reply:
I mean TXH_NP_FS_FIFOSIZ

Reply:
It's only used on host usb it could be removed.